companyweb_base/views/companyweb_partner_view.xml

```diff
@@ -0,0 +1,270 @@
+        <field name="inherit_id" ref="base.view_partner_form" />
+        <field name="groups_id" eval="[(6, 0, [ref('cweb_view')])]" />
+        <field name="arch" type="xml">
+            <field name="country_id" position="after">
+                <p>
+                    <div>
+
+                    <field name="cweb_show_button_enhance" invisible="1" />
+                    <button
+                            type="object"
+                            icon="companyweb_base/static/img/logo-en.resized.png"
+                            name="button_companyweb_enhance"
+                            class="btn-sm btn-link "
+                            aria-label="Enhance"
+                            title="Enhance"
+                            attrs="{'invisible': [('cweb_show_button_enhance','=', False)]}"
+                        />
+                </div>
+                </p>
+
+            </field>
+            <notebook position="inside">
+                <field name="cweb_show_tab" invisible="1" />
+                <page
+                    string="Company Web"
+                    attrs="{'invisible': [('cweb_show_tab','=', False)]}"
+                >
+                    <img
+                        alt="Companyweb"
+                        src="companyweb_base/static/img/logo-en.png"
+                    />
+                    <field name="cweb_name_enable" invisible="1" />
+                    <field name="cweb_jur_form_enable" invisible="1" />
+                    <field name="cweb_companystatus_enable" invisible="1" />
+                    <field name="cweb_address_enable" invisible="1" />
+
+                    <field name="cweb_creditLimit_enable" invisible="1" />
+                    <field name="cweb_creditLimit_info_enable" invisible="1" />
+                    <field name="cweb_startDate_enable" invisible="1" />
+                    <field name="cweb_endDate_enable" invisible="1" />
+                    <field name="cweb_score_enable" invisible="1" />
+                    <field name="cweb_image_enable" invisible="1" />
+
+                    <field name="cweb_warnings_enable" invisible="1" />
+                    <field name="cweb_url_enable" invisible="1" />
+                    <field name="cweb_url_report_enable" invisible="1" />
+
+                    <field name="cweb_vat_liable_enable" invisible="1" />
+                    <field name="cweb_balance_data_enable" invisible="1" />
+
+                    <field name="cweb_balance_year_enable" invisible="1" />
+                    <field name="cweb_closed_date_enable" invisible="1" />
+                    <field name="cweb_equityCapital_enable" invisible="1" />
+                    <field name="cweb_average_fte_enable" invisible="1" />
+                    <field name="cweb_addedValue_enable" invisible="1" />
+                    <field name="cweb_turnover_enable" invisible="1" />
+                    <field name="cweb_result_enable" invisible="1" />
```

Comment:
nit: maybe for these numerical fields we should have a different name than `_enable`, to distinguish them from the regular `_enable` fields, since the semantics is different.